src/runtime/runtime-simd.cc

```diff
@@ -367,10 +401,29 @@ RUNTIME_FUNCTION(                               \
   CONVERT_ARG_CHECKED(SOURCE_TYPE, a, 0);                     \
                                                               \
   TARGET_TYPE::value_t result;                                \
-  for (int i = 0; i < SOURCE_TYPE::kLanes; i++) {             \
-    FUNCTION(a->getAt(i), &result.storage[i]);                \
+  if (#FUNCTION  == "BitsTo") {                               \
+    uint8_t *t1 =                                             \
+        reinterpret_cast<uint8_t*>(&(a->get().storage[0]));   \
+    uint8_t *t2 =                                             \
+        reinterpret_cast<uint8_t*>(&result.storage[0]);       \
+    CopyBytes<kSimdTypeSize>(t2, t1);                         \
```

Comment:
I suggest to get the number of bytes by `sizeof`. And please add a `CHECK` to make sure `TARGET_TYPE` and `SOURCE_TYPE` has same number of bytes.